.github/workflows/cleanup-test-env.yml

```diff
@@ -41,3 +41,21 @@ jobs:
           MONGODB_ATLAS_OPS_MANAGER_URL: ${{vars.MONGODB_ATLAS_BASE_URL}}
           PROJECT_TO_NOT_DELETE: ${{vars.MONGODB_ATLAS_PROJECT_ID_CLOUD_DEV_NETWORK}}
         run: ./scripts/cleanup-test-env.sh
+  cleanup-test-env-qa:
```

Comment:
[nit] Why didn't we use a similar logic here to support qa via `atlas_cloud_env`?

Reply:
cleanup tasks are taking seconds to run (https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/7348484765) so saw no concern in having it run on a daily basis.